cg/utils/get_genome_build.py

```diff
@@ -0,0 +1,19 @@
+"""Helper functions related to the genome build."""
```

Comment:
I think we should place this in a utils folder in cg/meta/workflow. If the folder does not exist, feel free to create it